### PR TITLE
Add push gateway

### PR DESCRIPTION
**Pull request by BarryPiccinni**
The team have a need to push the output of ephemeral jobs to Prometheus.

This PR adds the pushgateway to the DEV environment to facilitate this.

### CODE REVIEW

**Review by mogaal — DISMISSED** [review]

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/claim-criminal-injuries-compensation-dev/resources/versions.tf [hunk 1]
```diff
@@ -1,4 +1,3 @@
-
 terraform {
-  required_version = ">= 0.12"
-}
+  required_version = "0.12.17"
```
Comment (jasonBirchall) [inline namespaces/live-1.cloud-platform.service.justice.gov.uk/claim-criminal-injuries-compensation-dev/resources/versions.tf:2]: I would leave the required version at ">=0.12" for now. The pipeline uses 0.12.30 and we're in the process of a migration (be it a little slow).

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/claim-criminal-injuries-compensation-dev/resources/pushgateway.tf [hunk 2]
```diff
@@ -0,0 +1,6 @@
+module "pushgateway" {
+  source = "github.com/ministryofjustice/cloud-platform-terraform-pushgateway?ref=1.2"
```
Comment (jasonBirchall) [inline namespaces/live-1.cloud-platform.service.justice.gov.uk/claim-criminal-injuries-compensation-dev/resources/pushgateway.tf:2]: Can you bump this down to 1.1. 1.2 contains the necessary code for 0.13. Really sorry for the confusion here.

**Review by jasonBirchall — CHANGES_REQUESTED** [review]
Hi Barry, there are a couple of tweaks that are completely our fault. Everything else looks good.

**Comment by BarryPiccinni** [comment]
@jasonBirchall Hi Jason, thanks for the review, that was really helpful. I was messing with the versions after I got an error in the build, but actioning the above has fixed it.

**Review by jasonBirchall — APPROVED** [review]

**Pull request merged**